### ISSUE TITLE
Test 4: Tables - headings in wrong list items

### ISSUE THREAD

**Issue by jasonkiss** [GOVTNZ/web-standards#8]
The list of steps for testing include a number of H5 headings, but each of these occurs in the list item preceding the list item to which it is a heading.

For example,

```
<li>
        <p>If none of these options have been used to markup the header cell, record a failure, noting the reason.</p>

        <h5>Layout tables</h5>
</li>
```

The "Layout tables" heading is actually for the next list item in the list.

**Comment by jasonkiss** [comment]
Same issue exists with H5 headings in the Keyboard test.

Possible solutions include:
* Moving the H5s in between discrete OLs, which is the approach taken for the H5 headings in the Captions and Transcripts test. The problem here is that each OL always starts at #1, because you can't use `@start` in MD.
* Remove the headings
* Rewrite the OLs in HTML instead of MD, and put the H5s before the relevant OL, and use `@start` to continue numbering across the OLs

**Comment by kbarcham** [comment]
Fixed and published.
